TestControllers/TestController.psm1

```diff
@@ -772,75 +804,77 @@ Class TestController
-					-downloadTo $BootLogDir -username $global:user -password $global:password | Out-Null
-				Write-LogInfo "$($VM.RoleName): $status kernel log, ${currentBootLogFile}, collected successfully."
-
-				Write-LogInfo "Checking for call traces in kernel logs.."
-				$KernelLogs = Get-Content $currentBootLog
-				$callTraceFound  = $false
-				foreach ($line in $KernelLogs) {
-					if (( $line -imatch "Call Trace" ) -and ($line -inotmatch "initcall ")) {
-						Write-LogErr $line
-						$callTraceFound = $true
-					}
-					if ($callTraceFound) {
-						if ($line -imatch "\[<") {
+					Run-LinuxCmd -ip $VM.PublicIP -port $VM.SSHPort -runAsSudo `
+						-username $global:user -password $global:password `
+						-command "dmesg > ./${currentBootLogFile}" | Out-Null
+					Copy-RemoteFiles -download -downloadFrom $VM.PublicIP -port $VM.SSHPort -files "./${currentBootLogFile}" `
+						-downloadTo $BootLogDir -username $global:user -password $global:password | Out-Null
+					Write-LogInfo "$($VM.RoleName): $status kernel log, ${currentBootLogFile}, collected successfully."
+
+					Write-LogInfo "Checking for call traces in kernel logs.."
+					$KernelLogs = Get-Content $currentBootLog
+					$callTraceFound  = $false
+					foreach ($line in $KernelLogs) {
+						if (( $line -imatch "Call Trace" ) -and ($line -inotmatch "initcall ")) {
 							Write-LogErr $line
+							$callTraceFound = $true
+						}
+						if ($callTraceFound) {
+							if ($line -imatch "\[<") {
+								Write-LogErr $line
+							}
 						}
 					}
-				}
-				if (!$callTraceFound) {
-					Write-LogInfo "No kernel call traces found in the kernel log"
-				}
+					if (!$callTraceFound) {
+						Write-LogInfo "No kernel call traces found in the kernel log"
+					}
 
-				if($status -imatch "Final") {
-					$ret = Compare-OsLogs -InitialLogFilePath $InitialBootLog -FinalLogFilePath $currentBootLog -LogStatusFilePath $KernelLogStatus `
-						-ErrorMatchPatten "fail|error|warning"
+					if($status -imatch "Final") {
+						$ret = Compare-OsLogs -InitialLogFilePath $InitialBootLog -FinalLogFilePath $currentBootLog -LogStatusFilePath $KernelLogStatus `
+							-ErrorMatchPatten "fail|error|warning"
 
-					if ($ret -eq $false) {
-						$retValue = $false
-					}
+						if ($ret -eq $false) {
+							$retValue = $false
+						}
 
-					# Removing final dmesg file from logs to reduce the size of logs.
-					# We can always see complete Final Logs as: Initial Kernel Logs + Difference in Kernel Logs
-					Remove-Item -Path $currentBootLog -Force | Out-Null
+						# Removing final dmesg file from logs to reduce the size of logs.
+						# We can always see complete Final Logs as: Initial Kernel Logs + Difference in Kernel Logs
+						Remove-Item -Path $currentBootLog -Force | Out-Null
 
-					Write-LogInfo "$($VM.RoleName): $status Kernel logs collected and compared successfully"
+						Write-LogInfo "$($VM.RoleName): $status Kernel logs collected and compared successfully"
```

Comment:
Indent

Reply:
Test Controller using 'tab' as indent, and already consistent with previous indenting, no need to update here.